_posts/2017-11-21-bio-formats-5-7-2.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: Release of Bio-Formats 5.7.2
+intro-blurb: The OME team is pleased to announce the release of Bio-Formats 5.7.2
+---
+Today we are releasing Bio-Formats 5.7.2 which includes the following changes:
+
+File format fixes and improvements:
+
+* Nikon ND2
+    * fixed a bug which would use the incorrect channel count for small-sized
+      single channel images
+
+* MetaMorph TIFF
+    * changed the reader's behavior to populate exposure times for all planes
+      when only a single exposure time is defined
+
+* DeltaVision
+    * improved parsing of the associated log files to add additional key value 
+      pairs to global metadata
+
+* EPS (Encapsulated PostScript)
+    * fixed an exception when reading pixel data in cases with embedded TIFF
+
+* GIF
+    * fixed a bug to display the correct data when reading planes out of order
+
+Bug fixes and improvements:
+
+* fixed failures with Ant build from a clean Maven repository by updating
+  Maven repositories to use HTTPS rather than HTTP
+* now using safe version checking for Bio-Formats plugins to prevent a bug
+  with Java 9
+* updated the JPEG-XR codec to allow either interleaved or non-interleaved
+  data to be returned
+
+Documentation improvements:
+
+* added clarification regarding Bio-Formats version requirements for using
+  Java 7 or above
+* updated download links to latest Bio-Formats release version
+* updated the link to the most active fork of JAI ImageIO
+* fixed a number of external broken links
+* added a Trello link for contributing external developers
+* added a link to [Adding format/reader documentation pages](https://docs.openmicroscopy.org/latest/bio-formats/developers/format-documentation.html)
+  to help those contributing to the documentation or supported formats pages
+* the [Bio-Rad Gel page](https://docs.openmicroscopy.org/latest/bio-formats/formats/bio-rad-gel.html) has been updated to add a link to biorad1sc_reader, an
+  external Python implementation (thanks to Matthew Clapp)
```

Comment:
we will probably need to add an entry to https://www.openmicroscopy.org/contributors/

Reply:
I've done that on this PR already

Reply:
sorry the changes were hidden being another window
Great thanks